core/domain/suggestion_registry.py

```diff
@@ -503,6 +503,10 @@ def __init__( # pylint: disable=super-init-not-called
         self.author_id = author_id
         self.final_reviewer_id = final_reviewer_id
         self.change = question_domain.QuestionChange(change)
+        # Update question_state_data_schema_version here instead of surfacing the
+        # version in the frontend.
+        self.change.question_dict['question_state_data_schema_version'] = (
```

Comment:
Would you need to change the corresponding tests too, to include this line?

Reply:
The tests are already checking for the schema version?

Reply:
I am not sure, I was just checking with you.